doc/conf.py

```diff
@@ -158,6 +158,8 @@
         "%matplotlib inline\n"
         "from pyvista import set_plot_theme\n"
         "set_plot_theme('document')\n"
+        # Reset all values of wrapping `vtkPolyData` in examples
```

Comment:
This is here because we need to guarantee that we reset the wrapping, right?

Reply:
Let's also reset the values of `_wrapper` in the example and make a note of it somewhere.

Reply:
This is the only clean way I could figure out how to reset the wrapping.  It seems that the same `pyvista` import is shared amongst all the examples, similar to what happened when running `pytest`.  Just resetting the value of `_wrapper` in the example runs the risk of an error occuring in the middle of the example, and then this leaking out into all subsequent examples.

I think this usage is for when users **only** want to use subclasses, so resetting it will be uncommon in practice.  I will read again to make sure this is clear.  It makes sense to put in the reset of `_wrapper`, again to make this point clear.

Reply:
We had addressed this in the testing by using a `try....finally` block, but this ruins the flow of the example with the interspersed RST blocks.  I also added this example at the end as a suggestion.